hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ReopenTableRegionsProcedure.java

```diff
@@ -234,5 +238,12 @@ protected void deserializeStateData(ProcedureStateSerializer serializer) throws
     tableName = ProtobufUtil.toTableName(data.getTableName());
     regions = data.getRegionList().stream().map(ProtobufUtil::toRegionLocation)
       .collect(Collectors.toList());
+    if (CollectionUtils.isNotEmpty(data.getRegionNamesList())) {
```

Comment:
Ditto here? And please use collect(Collectors.toList()), instead of creating an ArrayList?

Reply:
Sure, will use collect here.

Reply:
Done, updated with `collect`.